<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySecondQt_VSClass</class>
 <widget class="QMainWindow" name="MySecondQt_VSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySecondQt_VS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="origin_img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray_img">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="black_img">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>灰度图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>黑白图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_img">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>获取图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_gray_img">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存灰度图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_black_img">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存黑白图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_all_img">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>440</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>显示灰度图和黑白图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>灰度图和黑白图</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MySecondQt_VS.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>get_img</sender>
   <signal>clicked()</signal>
   <receiver>MySecondQt_VSClass</receiver>
   <slot>img_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_gray_img</sender>
   <signal>clicked()</signal>
   <receiver>MySecondQt_VSClass</receiver>
   <slot>save_gray()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_black_img</sender>
   <signal>clicked()</signal>
   <receiver>MySecondQt_VSClass</receiver>
   <slot>save_black()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1055</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>1052</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_all_img</sender>
   <signal>clicked()</signal>
   <receiver>MySecondQt_VSClass</receiver>
   <slot>show_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>img_clicked()</slot>
  <slot>save_gray()</slot>
  <slot>save_black()</slot>
  <slot>show_all()</slot>
 </slots>
</ui>
